lib/msf/core/data_store.rb

```diff
@@ -168,10 +168,13 @@ def to_s(delim = ' ')
     return str
   end
 
+  # Override Hash's to_h method so we can include the original case of each key
+  # (failing to do this breaks a number of places in framework and pro that use
+  # serialized datastores)
```

Comment:
This brings back the dropped comment that was previously here:
https://github.com/rapid7/metasploit-framework/blame/fd07da3519fb59636217ad220a381b6c32d3bf3b/lib/msf/core/data_store.rb#L70-L73